bundles/de.uka.ipd.sdq.simulation.abstractsimengine.desmoj/src/de/uka/ipd/sdq/simulation/abstractsimengine/desmoj/DesmoJExperiment.java

```diff
@@ -21,8 +22,14 @@ public DesmoJExperiment(final DesmoJModel model) {
         super(model);
 
         this.desmojModel = model;
-        this.experiment = new Experiment(model.getConfiguration().getNameExperimentRun(), TimeUnit.NANOSECONDS,
-                TimeUnit.SECONDS, new SingleUnitTimeFormatter(TimeUnit.SECONDS, TimeUnit.NANOSECONDS, 9, false));
+        this.experiment = new Experiment(model.getConfiguration().getNameExperimentRun(), ".", //<- Path for storing output that defaults to .

```

Comment:
I was wondering whether it would make sense to provide the path to a temporary folder (like Files.createTempDirectory). On the other hand we'd create temp folders without ever putting something in. I am not sure what is the better option.

Reply:
if I'm correct this String is never used, so creating a temp dir would just be a waste